### ISSUE TITLE
setting "basepath: .."in jsTestDriver.conf causes UnreadableFilesException

### ISSUE THREAD

**Issue by GoogleCodeExporter** [manojjsm/js-test-driver#356]
```
What steps will reproduce the problem?
1. Move jsTestDriver.conf into a subdirectory named "config"
2. set basepath: .. in jsTestDriver.conf
3. Run the tests

What is the expected output? 
Tests should run without any UnreadableFilesException errors.

What do you see instead?
Tests run correctly, but UnreadableFilesException is produced because files 
continue to be searched for under "config" directory, despite basepath being 
set to ".."

What version of the product are you using?
RubyMine 4.0.1 with jsTestDriver plugin 
Version 1.4, 2012 Febrary 21 (build on JsTestDriver-1.3.4-a)

On what operating system?
Windows 7

Please provide any additional information below.
7:32:06 PM UnreadableFilesException: 
Cannot read [ 
// C:\RubymineProjects\project\config\test\lib\jasmine\jasmine.js 
// ] derived from test/lib/jasmine/jasmine.js 
// Cannot read [ 
// 
C:\RubymineProjects\project\config\test\lib\jasmine-jstd-adapter\JasmineAdapter.
js 
// ] derived from test/lib/jasmine-jstd-adapter/JasmineAdapter.js
etc....
```

Original issue reported on code.google.com by `dan...@gmail.com` on 2 Apr 2012 at 11:45

**Comment by GoogleCodeExporter** [comment]
```
In addition, does not strip file system root off of project lines,
and does not enable hyperlinks in console to navigate to proper source location.
```

Original comment by `dan...@gmail.com` on 2 Apr 2012 at 11:53

**Comment by GoogleCodeExporter** [comment]
```
I've successfully reproduced it.
```

Original comment by `Sergey.S...@gmail.com` on 3 Apr 2012 at 6:51
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Is this an issue in the JsTD library?
```

Original comment by `corbinrs...@gmail.com` on 3 Apr 2012 at 6:57

**Comment by GoogleCodeExporter** [comment]
```
No, that is plugin's fault - it parses config file to enable hyperlinks in 
console for navigation purpose.
Fixed version of the plugin will be uploaded in a week or two.
```

Original comment by `Sergey.S...@gmail.com` on 4 Apr 2012 at 12:00
- Added labels: **IntelliJ**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 117.222 version of the plugin (compatible with RubyMine 4.0.3).
```

Original comment by `Sergey.S...@gmail.com` on 18 Apr 2012 at 5:49
- Changed state: **Fixed**
